src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -365,9 +389,14 @@ export default {
 						_each(this.categories, category => {
 							ssrLoanIds.push(category.id);
 						});
-						const loanChannels = _get(data, 'lend.loanChannelsById');
-						const filteredLoanChannels = loanChannels.filter(loan => {
-							return !ssrLoanIds.includes(loan.id);
+						const loanChannels = this.categoryServiceExpVersion === 'b'
```

Comment:
Might be worth putting `this.categoryServiceExpVersion === 'b'` in a computed value. It is used a decent amount, including in the template.